feature-proposals.rst

```diff
@@ -30,111 +32,166 @@ a new Wiki page containing an RFC with a unique title MUST be created. In this
 
-RFC authors SHOULD NOT announce the voting intent when the RFC discussion is
-still ongoing and SHOULD consider all new discussion points that are brought
-forward. RFC authors SHOULD also consider all new discussion points brought
-forward after announcing the voting intent before actually proceeding with
-starting the vote. If objective improvements are brought forward after
-announcing the voting intent they SHOULD NOT be disregarded in the interest of
-time and at the expense of quality.
+If there are any secondary votes included, the RFC text describing the vote MUST include
+whether the secondary vote is plurality or Single-Transferrable Vote, how ties will be handled,
+whether it is a 2/3 majority Yes/No or a 50% threshold, and any other pertinent information
+to ensure the implications of the secondary vote are clearly understood.
 
-If major or minor changes to the RFC text are made after announcing the voting
-intent, the vote MUST NOT be started. Instead the cooldown period MUST be reset
-and a new voting intent MUST be announced when the RFC is ready for voting after
-the new cooldown period has elapsed.
+=================
+Voting procedure
+=================
 
 The actual start of the vote MUST be announced on the mailing list in a separate
-thread with a ``[VOTE]`` prefix followed by the RFC title as the Subject. The
-email MUST include a link to the Wiki page of the RFC and to the mailing list
-archives of the discussion thread. The link to the mailing list archives of the
-voting thread MUST be added to the RFC as soon as possible and no later than the
-announcement of the results of the vote.
-
-The email MUST furthermore clearly specify the voting formalities, such as the
-number of votes that can be cast, the interpretation of the voting results and
-the end date of the voting. The end date MUST be specified with minute-precision
-and is not part of the voting period, all votes must be cast before the
-specified end date. The voting period MUST be open for at least two weeks (336
-hours). The voting period MAY be extended as necessary (e.g. to accommodate
-holiday periods).
+thread with a ``[VOTE]`` prefix followed by the RFC title as the Subject.
+
+The voting announcement email MUST include:
+
+- A link to the Wiki page of the RFC.
+- A link to the discussion thread.
+- A notice if there are multiple votes to cast, as each requires a separate form submission.
+- The exact date and time the vote will end, with minute precision and timezone.
+
+The end of the vote MUST additionally be specified in the voting widget on the
+RFC so that it will auto-close at the specified timestamp.
+
+The link to the mailing list archives of the voting thread MUST be added to
+the RFC as soon as possible and no later than the announcement of the results of the vote.
+
+The voting period MUST be at least two weeks (336 hours), but MAY be up to
+four weeks if necessary to avoid conflict with well-known holiday periods.
```

Comment:
I introduced a 4 week cap here, as it seems prudent to not allow a 9 month vote or something.